docs-developer/deploying.md

```diff
@@ -40,16 +40,39 @@ the [section related to this step](#how-to-merge-l10n-into-main) for more detail
 After merging the `l10n` branch, we can continue with the deployment.
 The easiest by far is to
 [create a pull request on GitHub](https://github.com/firefox-devtools/profiler/compare/production...main?expand=1).
-It would be nice to write down the main changes in the PR description.
+It would be nice to write down the main changes in the PR description ([see below](#user-content-helpful-git-commands-to-write-the-main-changes)).
 
 After the PR is created all checks should run. When it's ready the PR can be
 merged. Be careful to always use the **create a merge commit** functionality,
-not _squash_ or _rebase_, to keep a better history.
+not _squash_ or _rebase_, to keep a better history. Also you can copy the PR
+description as the commit log body, so that the changelog is also present in the
+git repository.
 
 Once it's done the new version should be deployed automatically. You can follow the
 process on [Netlify's dashboard](https://app.netlify.com/sites/perf-html/deploys)
 if you have access.
 
+### Helpful git commands to write the main changes
+
+Here is how you can gather the changes since the last deploy:
+
+1. Gather all the code changes:
+
+```
+git fetch upstream && git log upstream/production..upstream/main --first-parent --oneline --no-decorate --format="format:[%an] %s"
```

Comment:
Nice, I was also using a similar git command to generate the list, but it wasn't looking as fancy as this one.

One thing I did in my command though was to add `| tail -r` to the end of the command to reverse the lines, so I can keep the chronological order of the commits. But looks like `-r` on tail only exist on mac and not linux. Looked at the `git log` documentation again and looks like it actually has `--reverse`. But I don't know if this is something we should really do or not, let me know what you think :)

Reply:
`--reverse` sounds great, I'll add it, thanks for the addition.

Be careful with your use of `tail` because by default this displays just 10 lines! I think you really wanted `tac`.

Reply:
Actually `tail -r` on macos works a bit differently. If the `-r ` flag is set it defaults to outputting all the input lines: https://ss64.com/mac/tail.html This is actually a macos only flag and it doesn't seem to exist on linux. There is no `tac` on macos :(